plugins/network-elements/bigswitch/src/com/cloud/network/bigswitch/BigSwitchBcfApi.java

```diff
@@ -296,7 +296,7 @@ private void setHttpHeader(final HttpMethodBase m) {
         m.setRequestHeader(CONTENT_TYPE, CONTENT_JSON);
         m.setRequestHeader(ACCEPT, CONTENT_JSON);
         m.setRequestHeader(HTTP_HEADER_INSTANCE_ID, CLOUDSTACK_INSTANCE_ID + "-" + zoneId);
-        if (hash != "" ) {
+        if (! "".equals(hash)) {
```

Comment:
sure but why two calls (isNotEmpty calls isEmpty) while this is all we need? Don't mistake my question for sarcasm. I want to know.